networkx/algorithms/richclub.py

```diff
@@ -75,7 +75,8 @@ def rich_club_coefficient(G, normalized=True, Q=100, seed=None):
             "rich_club_coefficient is not implemented for graphs with self loops."
         )
     rc = _compute_rc(G)
-    if normalized:
+    # need at least 4 nodes to perform a double edge swap
```

Comment:
I don't think you need to check `G.number_of_nodes() >= 4` here. Currently, the `nx.double_edge_swap` raises a  `NetworkXError: Graph has fewer than four nodes.` for this case.

```.py
G = nx.Graph()
G.add_nodes_from([0,1,2])
G.add_edges_from([(1,2),(2,0)])
print(nx.rich_club_coefficient(G))
 
# output : 
# earlier : NetworkXError: Graph has fewer than four nodes.
# now : {0: 0.6666666666666666}
```